modules/apps/object/object-test/src/testFunctional/paths/ObjectAdmin.path

```diff
@@ -534,18 +533,11 @@
 	<tr>
 		<td>VIEW_STATE_NAME</td>
 		<td>//h3[(@class='lfr-objects__card-title') and text()='Select the State Flow']//..//..//div[contains(@class,'lfr-objects__card-title--with-padding') and contains(.,'${key_stateName}')]</td>
-=======
-		<td>SELECT_CHOOSE_OPTION</td>
-		<td>//option[contains(text(),'${chooseOption}')]</td>
 		<td></td>
 	</tr>
 	<tr>
-		<td>INPUT_METHOD</td>
-<<<<<<< HEAD
-		<td>//input[contains(@class, 'custom-control-input') and contains(@type,'checkbox')]/..//span[contains(text(),'${inputMethod}')]</td>
->>>>>>> a02f5cb (LPS-156563 Add ObjectActions#CanEditActionName)
-=======
-		<td>//input[contains(@class, 'custom-control-input') and contains(@type, 'checkbox')]//..//span[contains(text(), '${key_inputMethod}')]</td>
+		<td>SELECT_CHOOSE_OPTION</td>
+		<td>//option[contains(text(),'${chooseOption}')]</td>
 		<td></td>
```

Comment:
In the future, if there are ugly commit mistakes like this, it's better to push it back to the original author and ask them to fix their commit first.